src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -97,7 +95,7 @@ void EnTa_SetupAction(EnTa* this, EnTaActionFunc actionFunc, EnTaAnimFunc animFu
 }
 
 void EnTa_SetTextForTalkInLonLonHouse(EnTa* this, PlayState* play) {
-    u16 faceReaction = Text_GetFaceReaction(play, TALON_FACE_REACTION_SET);
+    u16 textId = Text_GetFaceReaction(play, FACE_REACTION_SET_TALON);
```

Comment:
I think faceReactionTextId would be better here since the logic is less plain than in other actors